tsconfig.json

```diff
@@ -2,6 +2,17 @@
   "include": ["index.js"],
   "compilerOptions": {
     "allowJs": true,
+    "checkJs": true,
+    "skipLibCheck": true,
+    "lib": ["es2021"],
+    "target": "es2021",
```

Comment:
I don't remember what ES version is supported by Node.js 16. We can downgrade these two lines to `es2020` if we run into runtime errors not caught by the compiler.